src/sage/groups/braid.py

```diff
@@ -1622,19 +1659,43 @@ def conjugating_braid(self, other):
             sage: a = B([2, 2, -1, -1])
             sage: b = B([2, 1, 2, 1])
             sage: c = b * a / b
-            sage: d = a.conjugating_braid(c)
-            sage: d * c / d == a
-            True
-            sage: d
+            sage: d1 = a.conjugating_braid(c)
+            sage: print (d1)
             s1*s0
-            sage: d * a / d == c
+            sage: d1 * c / d1 == a
+            True
+            sage: d1 * a / d1 == c
+            False
+            sage: l = sage.groups.braid.conjugatingbraid(a,c)
+            sage: d1 == B._element_from_libbraiding(l)
+            True
+            sage: b = B([2, 2, 2, 2, 1])
+            sage: c = b * a / b
+            sage: d1 = a.conjugating_braid(c)
+            sage: print (len(d1.Tietze()))
```

Comment:
PEP8
```suggestion
            sage: print(len(d1.Tietze()))
```
and similar below.

Reply:
Done